model/device.go

```diff
@@ -131,11 +132,11 @@ func (d *Device) UnmarshalBSON(b []byte) error {
 		return err
 	}
 	for _, attr := range d.Attributes {
-		if attr.Scope == "identity" {
+		if attr.Scope == "identity" && attr.Name == "group" {
```

Comment:
why arent we using `AttrScopeIdentity` in here?
also: `AttrGroup = "group"` or something along these lines wouldnt be good?

Reply:
I changed it to AttrScopeSystem.